LTSupportAutomotive/LTOBD2PID.m

```diff
@@ -90,6 +91,20 @@ -(instancetype)initWithString:(NSString*)string inFreezeFrameDTC:(LTOBD2DTC*)fre
 #pragma mark -
 #pragma mark API for subclasses
 
+-(NSNumber*) decodeSingleByteDoubleValueWithOffset:(double)offset factor:(double)factor
```

Comment:
Minor nitpick: Coding style on this library is to not have whitespace between the result type and the start of the method name.

Reply:
No problem, happy to align with coding style of the library. Consistency is important, I will fix where I see it.

Reply:
Ok, think i got all of them